README.md

```diff
@@ -44,25 +44,92 @@ offering features such as:
+
+```shell
+pip install advanced-alchemy
+```
+
 > [!IMPORTANT]\
 > Check out [the installation guide][install-guide] in our official documentation!
 
-### Litestar
+### Repositories
+
+Advanced Alchemy includes a set of asynchronous and synchronous repository classes for easy CRUD operations on your SQLAlchemy models.
 
 ```python
-from advanced_alchemy.extensions.litestar.plugins import SQLAlchemyPlugin
-from advanced_alchemy.extensions.litestar.plugins.init.config import SQLAlchemyAsyncConfig
+from advanced_alchemy.base import UUIDBase
+from advanced_alchemy.repository import SQLAlchemySyncRepository
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Mapped, Session, sessionmaker
+
+class User(UUIDBase):
+    # you can optionally override the generated table name by manually setting it.
+    __tablename__ = "user_account"  # type: ignore[assignment]
+    email: Mapped[str]
+    name: Mapped[str]
+
+class UserRepository(SQLAlchemySyncRepository[User]):
+    """User repository."""
+
+    model_type = User
+
+# use any compatible sqlalchemy engine.
+engine = create_engine("duckdb:///:memory:")
+session_factory = sessionmaker(engine, expire_on_commit=False)
+
+with session_factory() as db_session:
+  repo = UserRepository(session=db_session)
+  bulk_users = [
+    {email: 'cody@advanced-alchemy.dev','name': 'Cody'},
+    {email: 'janek@advanced-alchemy.dev','name': 'Janek'},
+    {email: 'peter@advanced-alchemy.dev','name': 'Peter'},
+    {email: 'jacob@advanced-alchemy.dev','name': 'Jacob'}
+  ]
+  objs = repo.add_many([User(**raw_user) for raw_user in bulk_users])
+  db_session.commit()
+  print(f"Created {len(objs)} new objects.")
+
+  # 2) Select paginated data and total row count.  Pass additional filters as kwargs
+  created_objs, total_objs = repo.list_and_count(name="Cody", LimitOffset(limit=10, offset=0))
+  print(f"Selected {len(created_objs)} records out of a total of {total_objs}.")
+
+  # 3) Let's remove the batch of records selected.
+  deleted_objs = repo.delete_many([new_obj.id for new_obj in created_objs])
+  print(f"Removed {len(deleted_objs)} records out of a total of {total_objs}.")
+
+  # 4) Let's count the remaining rows
+  remaining_count = repo.count()
+  print(f"Found {remaining_count} remaining records after delete.")
+```
 
-from litestar import Litestar
+For a full standalone example, see the sample [here][standalone-example]
+
+### Web Frameworks
+
+Advanced Alchemy works with nearly all Python web frameworks. Several helpers for popular libraries are included, and additional PRs to support others are welcomed.
 
-plugin = SQLAlchemyPlugin(config=SQLAlchemyAsyncConfig(connection_string="sqlite+aiosqlite:///test.sqlite"))
+#### Litestar
 
+Advanced Alchemy is the official SQLAlchemy integration for Litsetar.
 
-app = Litestar(plugins=[plugin])
+In addition to installed with `pip install advanced-alchemy`, It can be installed installed as a Litestar extra with `pip install litestar[sqlalchemy]`.
```

Comment:
```suggestion
In addition to installed with `pip install advanced-alchemy`, it can also be installed installed as a Litestar extra with `pip install litestar[sqlalchemy]`.
```